addons/fleet/models/fleet_vehicle.py

```diff
@@ -77,7 +77,7 @@ def _get_default_state(self):
     @api.depends('model_id', 'license_plate')
     def _compute_vehicle_name(self):
         for record in self:
-            record.name = record.model_id.brand_id.name + '/' + record.model_id.name + '/' + record.license_plate
+            record.name = record.model_id.brand_id.name + '/' + record.model_id.name + '/' + (record.license_plate or 'No Plate')
```

Comment:
`_('No License Plate')` to be translatable

Reply:
:+1: